funcake_villanova_dag.py

```diff
@@ -21,12 +21,20 @@
 # {
 #   "endpoint": "http://digital.library.villanova.edu/OAI/Server",
 #   "included_sets": ["dpla"],
-#   "excluded_sets": [],
-#   "md_prefix": "oai_dc"
+#   "excluded_sets": [], <--- OPTIONAL
+#   "md_prefix": "oai_dc",
+#   "xsl_branch": "my_test_branch", <--- OPTIONAL
+#   "xsl_filename": "transforms/my_test_transform.xml", <--- OPTIONAL
+#   "schematron_filename": "validations/test_validation", <--- OPTIONAL
 # }
-MDX_PREFIX = VILLANOVA_OAI_CONFIG.get("md_prefix")
+MDX_PREFIX   = VILLANOVA_OAI_CONFIG.get("md_prefix")
 INCLUDE_SETS = VILLANOVA_OAI_CONFIG.get("included_sets")
 OAI_ENDPOINT = VILLANOVA_OAI_CONFIG.get("endpoint")
+XSL_BRANCH   = VILLANOVA_OAI_CONFIG.get("xsl_branch", "master")
+XSL_FILENAME = VILLANOVA_OAI_CONFIG.get("xsl_filename", "transforms/villanova.xsl")
+XSL_REPO     = VILLANOVA_OAI_CONFIG.get("xsl_repo", "tulibraries/aggregator_mdx")
+SCHEMATRON_FILENAME = VILLANOVA_OAI_CONFIG.get("schematron_filename", "validations/padigital_reqd_fields.sch")
```

Comment:
Wondering if we should have a separate `VILLANOVA_XSLT_CONFIG` for two reasons: 
1. will work with CSV harvests
2. we can attach different filtering schematrons to both the HARVEST and the XSLT tasks.

Reply:
@cmharlow Yeah, I was wondering about that. I'll make that change

Reply:
thanks @bibliotechy